### PR TITLE
Fix: Contact Buttons - fix to padding, location and chat bubble

### PR DESCRIPTION
**Pull request by mserino**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

File: modules/conversion-center/assets/scss/widgets/contact-buttons.scss [hunk 1]
```diff
@@ -442,6 +442,7 @@
 		font-size: 20px;
 		line-height: 25px;
 		font-weight: 600;
+		margin-bottom: 0;
```
Comment (louiswol94) [inline modules/conversion-center/assets/scss/widgets/contact-buttons.scss:445]:
```suggestion
		margin-block-end: 0;
```

File: modules/conversion-center/base/widget-contact-button-base.php [hunk 2]
```diff
@@ -1587,6 +1590,10 @@ protected function add_style_contact_section(): void {
 					'type' => Controls_Manager::DIMENSIONS,
 					'size_units' => [ 'px', '%', 'em', 'rem' ],
 					'default' => [
+						'top' => '20',
+						'bottom' => '20',
+						'left' => '20',
+						'right' => '20',
```
Comment (louiswol94) [inline modules/conversion-center/base/widget-contact-button-base.php:1596]: Can we not keep it in the CSS and leave the control defaults?

Comment (mserino) [reply modules/conversion-center/base/widget-contact-button-base.php:1596]: @louiswol94 this was actually a request from Product, they want to have the control already populated by default.
Task here: https://tenup.teamwork.com/app/tasks/19514550

File: modules/conversion-center/assets/scss/widgets/contact-buttons.scss (hunk 1 above)
Comment (mserino) [reply modules/conversion-center/assets/scss/widgets/contact-buttons.scss]: done

**Review by louiswol94 — APPROVED** [review]

**Pull request merged**